benchmark/es/defaultparams-bench.js

```diff
@@ -5,7 +5,7 @@ const assert = require('assert');
 
 const bench = common.createBenchmark(main, {
   method: ['withoutdefaults', 'withdefaults'],
-  millions: [100]
+  n: [100e6]
```

Comment:
I'd like to use `1e8` for consistency. (ditto throughout)

Reply:
@starkwang so everywhere I should replace the value of n with 1e8?

Reply:
@juggernaut451 `100e6` means `100 * 1e6`, `1e6` means 10^6. Thus `100e6` can be written as `1e8`.

Reply:
changes made. Thanks